### PR TITLE
Added some docs and a directory with the proposed name.

### PR DESCRIPTION
**Pull request by goosemania**
It's the same directory as the existing one, it's just named
to follow the suggested convention.
The old one will be removed as a separate step, not to break
existing tests.

[noissue]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,70 @@
...
+The rest of identifiers which make it easier to figure out which snapshot is chosen can
+ be specified separated by the underscore.
+
+Examples:
+ * file_base_4repos
+ * file_changed_4repos_removal
+ * mix_base_rpm_container
+
+
+#### Config to generate a snapshot
```
Comment (goosemania) [inline README.md:50]: Is it worth automating the snapshot generation? or is it too much?

Comment (dralley) [reply README.md:50]: Probably not.  At least, not beyond the level of a simple shell script that does it.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,70 @@
 # pulp-2to3-migration-test-fixtures
 
-This repository contains a mongodump archives, /var/lib/pulp/content archives, and /var/lib/published archives.
+This repository contains Pulp 2 snapshots in the form of mongodump archives, /var/lib/pulp/content
+ archives, and /var/lib/pulp/published archives to used in tests for the pulp-2to3-migration plugin.
```
Comment (ipanova) [inline README.md]: s/to used/to be used

File: README.md [hunk 3]
```diff
@@ -1,8 +1,70 @@
...
 
-The following commands were used to generated the current snapshots. 
+### Structure
+
+There is a separate directory for each Pulp 2 snapshot.
+Each directory has to have:
+ - a mongodump
+ - a /var/lib/pulp/content snapshot
+ - a /var/lib/pulp/published snapshot
+
```
Comment (ipanova) [inline README.md:15]: - a config to generate the Pulp 2 snapshot

File: README.md [hunk 4]
```diff
@@ -1,8 +1,70 @@
...
+### Structure
+
+There is a separate directory for each Pulp 2 snapshot.
+Each directory has to have:
+ - a mongodump
+ - a /var/lib/pulp/content snapshot
+ - a /var/lib/pulp/published snapshot
+ 
+Each directory should have a config to generate the Pulp 2 snapshot.
+It helps to regenerate it if needed and it serves as a documentation of what is present in the
```
Comment (ipanova) [inline README.md]: s/It/Config helps

Comment (ipanova) [reply README.md]: s/it/to regenerate the snapshot

File: README.md (hunk 1 above)
Comment (ipanova) [reply README.md:50]: it makes sense to provide a shell script

File: README.md [hunk 5]
```diff
@@ -11,10 +73,10 @@ The following commands were used to generated the current snapshots.
     sudo -u apache pulp-manage-db;
     sudo systemctl start httpd pulp_workers pulp_resource_manager pulp_celerybeat
     pulp-admin login -u admin -p admin
-    pulp-admin iso repo create --feed https://repos.fedorapeople.org/pulp/pulp/fixtures/file/ --repo-id file
-    pulp-admin iso repo create --feed https://repos.fedorapeople.org/pulp/pulp/fixtures/file2/ --repo-id file2 --download-policy on_demand
-    pulp-admin iso repo create --feed https://repos.fedorapeople.org/pulp/pulp/fixtures/file-many/ --repo-id file-many --download-policy on_demand
-    pulp-admin iso repo create --feed https://repos.fedorapeople.org/pulp/pulp/fixtures/file-large/ --repo-id file-large
+    pulp-admin iso repo create --feed https://fixtures.pulpproject.org/file/ --repo-id file
```
Comment (ipanova) [inline README.md:93]: :+1:

File: README.md (hunk 4 above)
Comment (goosemania) [reply README.md]: you are too specific ;), the word `it` is perfect, it covers anything one needs :P

File: README.md (hunk 3 above)
Comment (goosemania) [reply README.md:15]: it's not among mandatory ones, so it's mentioned in the sentence below.

File: README.md (hunk 1 above)
Comment (goosemania) [reply README.md:50]: I'll consider it, but not as  a part of this PR.

**Review by ipanova — APPROVED** [review]

**Review by dralley — APPROVED** [review]

**Pull request merged**